src/jit/SimdInl.h

```diff
@@ -311,3 +311,25 @@ static void emitGlobalSet128(sljit_compiler* compiler, Instruction* instr)
     sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_MEM1(SLJIT_R0), context->globalsStart + globalSet->index() * sizeof(void*));
     sljit_emit_simd_mov(compiler, SLJIT_SIMD_STORE | SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_128, src.arg, SLJIT_MEM1(SLJIT_R0), JITFieldAccessor::globalValueOffset());
 }
+
+static void emitSelect128(sljit_compiler* compiler, Instruction* instr)
+{
+    Operand* operands = instr->operands();
+    assert(instr->opcode() == ByteCode::SelectOpcode && instr->paramCount() == 3);
+
+    JITArg args[2];
+    JITArg target(operands + 3);
+    JITArg cond(operands + 2);
+    simdOperandToArg(compiler, operands, args[0], SLJIT_SIMD_ELEM_128, SLJIT_FR2);
```

Comment:
fr0/fr1 for most systems, and fr0/fr2 when `sljit_has_cpu_feature(SLJIT_SIMD_REGS_ARE_PAIRS)`

Reply:
Should be fixed now, I think.  Although... come to think of it, wouldn't it make sense to make the loading from memory to registers conditional on the branch?  Or is loading both parameters with `simdOperandToArg` acceptable?

Reply:
Done, but my question about optimizing away the loads still stands.  Wish I had remembered to ask it during today's meeting.